locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -364,7 +364,15 @@ public boolean isWindows(NodeMetadata node, ConfigBag config) {
                 ? (OsFamily.WINDOWS == os.getFamily()) 
                 : (OsFamily.WINDOWS == confFamily);
     }
-
+    
+    public boolean isNodeFirewalldEnabled(NodeMetadata node) {
+        String OS = node.getOperatingSystem().getFamily().toString();
+        String version = node.getOperatingSystem().getVersion();
+        return node.getOperatingSystem().getVersion().startsWith("7") &&
+             (node.getOperatingSystem().getFamily().equals(OsFamily.RHEL) ||
+              node.getOperatingSystem().getFamily().equals(OsFamily.CENTOS));
```

Comment:
The check is not future proof. Instead of checking fixed OS versions is it possible to check for the existence of firewall-cmd instead? This will allow the blueprint to work on future versions of the OS without any changes.